bin/generate

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'tempfile'
+require_relative '../generatorv2/lib/generator'
+
+parser = OptionParser.new
+
+parser.on('-v', '--version', 'Print the version') do
+  puts '0.1.0'
+end
+
+parser.on('-h', '--help', 'Prints help') do
+  puts parser
+end
+
+parser.on('-a', '--all', 'Generate all exercises') do
+  exercises = Dir.entries('./exercises/practice').select { |f| File.directory? File.join('./exercises/practice', f) }
+  exercises.each do |exercise|
+    if File.exist?("./exercises/practice/#{exercise}/.meta/test_template.erb")
+      Generator.new(exercise).generate
+    end
+  end
+end
+
+parser.on('--verify', 'Verify all exercises') do
+  exercises = Dir.entries('./exercises/practice').select { |f| File.directory? File.join('./exercises/practice', f) }
+  exercises.each do |exercise|
+    if File.exist?("./exercises/practice/#{exercise}/.meta/test_template.erb")
+      current_code = File.read("./exercises/practice/#{exercise}/#{exercise}_test.rb")
+      f = Tempfile.create
+      Generator.new(exercise).generate(f.path)
+      generated_code = f.read
+      raise RuntimeError.new("The result generated for: #{exercise}, doesnt match the current file") if current_code != generated_code
```

Comment:
```suggestion
      raise RuntimeError.new("The result generated for: #{exercise}, doesn't match the current file") unless current_code == generated_code
```

Typographical error (missing apostrophe) and preference of "positive conditional statement".

Prefer using `unless positive_conditional_statement` rather than `if negative`, this can keep all (or at least most) of our conditional statements positive.


We should also consider if `RuntimeError` is the best error here.  It could be `VerificationError` instead, which would be more specific.

We might also `fail` instead of `raise` for the communication that this is purposefully failed here, given the right conditions.